params/computation_cost_params.go

```diff
@@ -150,4 +150,6 @@ const (
 	FeePayerComputationCost             = 10
 	ValidateSenderPerSigComputationCost = 180000
 	ValidateSenderBaseComputationCost   = 10000
+	Blake2bBaseComputationCost          = 10000
+	Blake2bScaleComputationCost         = 100
```

Comment:
How did these values set? I cannot find them on eth's.

Reply:
@winnie-byun Computation costs are the one only used in Klaytn. @yoomee1313, could you please share the test or the method you used to derive these values?

Reply:
@winnie-byun thanks for the comment. I re-think about the value, and changed the `Blake2bScaleComputationCost` from 100 to 10. 
@ehnuje I referenced `ValidateSenderBaseComputationCost`(10000) to set `Blake2bBaseComputationCost`. Since Klaytn's computation cost limit(1,0000,0000) is 10 times larger than the Geth's block gas limit(1000,0000), I set `Blake2bScaleComputationCost` to 10. Those are the means of limiting execution, so, the scale value was set to 10 in order to multiply the computation cost by 10 times the gas price.

Reply:
@yoomee1313 Have you compared the performance of `ValidateSenderBaseComputationCost` and `Blake2bBaseComputationCost`? How did you come up with the idea of referencing `ValidateSenderBaseComputationCost`?

Reply:
I compared the performance with benchmark test function. I slightly changed `blake2f vector-7` input to make similar gas cost with validateSender test input.
In brief, blake2f test costs 5137 gas and the computation speed is 44262 ns/op. ValidateSender test costs 5000 gas and the computation speed is 48637 ns/op. According to the result, I think performance is similar, so I referenced validateSender's BaseComputationCost. 

```
BenchmarkPrecompiledBlake2F/vector_7-Gas=5137
BenchmarkPrecompiledBlake2F/vector_7-Gas=5137-12                      26664        44449 ns/op
```
```
BenchmarkPrecompiledValidateSender/validateSenderTest1-Gas=5000
BenchmarkPrecompiledValidateSender/validateSenderTest1-Gas=5000-12.   24156        48637 ns/op
```